core/common/assets/js/common.js

```diff
@@ -35,6 +36,8 @@ class ElementorCommonApp extends elementorModules.ViewModule {
 
 		this.api = new API();
 
+		this.debug = new Debug();
```

Comment:
Should be the first thing.

Reply:
Just did it, and then realize that the debug script depends on the "ajax" module that registered in "initModules" method.